floodviz/test/hydrograph_utils_tests.py

```diff
@@ -0,0 +1,84 @@
 "338", "qualifiers": ["A"], "dateTime": "2008-05-27T14:00:00.000-05:00"}, {"value": "336", "qualifiers": ["A"], "dateTime": "2008-05-27T14:15:00.000-05:00"}, {"value": "339", "qualifiers": ["A"], "dateTime": "2008-05-27T14:30:00.000-05:00"}, {"value": "341", "qualifiers": ["A"], "dateTime": "2008-05-27T14:45:00.000-05:00"}, {"value": "339", "qualifiers": ["A"], "dateTime": "2008-05-27T15:00:00.000-05:00"}, {"value": "341", "qualifiers": ["A"], "dateTime": "2008-05-27T15:15:00.000-05:00"}, {"value": "341", "qualifiers": ["A"], "dateTime": "2008-05-27T15:30:00.000-05:00"}, {"value": "339", "qualifiers": ["A"], "dateTime": "2008-05-27T15:45:00.000-05:00"}, {"value": "338", "qualifiers": ["A"], "dateTime": "2008-05-27T16:00:00.000-05:00"}, {"value": "334", "qualifiers": ["A"], "dateTime": "2008-05-27T16:15:00.000-05:00"}, {"value": "339", "qualifiers": ["A"], "dateTime": "2008-05-27T16:30:00.000-05:00"}, {"value": "336", "qualifiers": ["A"], "dateTime": "2008-05-27T16:45:00.000-05:00"}, {"value": "336", "qualifiers": ["A"], "dateTime": "2008-05-27T17:00:00.000-05:00"}, {"value": "338", "qualifiers": ["A"], "dateTime": "2008-05-27T17:15:00.000-05:00"}, {"value": "336", "qualifiers": ["A"], "dateTime": "2008-05-27T17:30:00.000-05:00"}, {"value": "339", "qualifiers": ["A"], "dateTime": "2008-05-27T17:45:00.000-05:00"}, {"value": "338", "qualifiers": ["A"], "dateTime": "2008-05-27T18:00:00.000-05:00"}, {"value": "336", "qualifiers": ["A"], "dateTime": "2008-05-27T18:15:00.000-05:00"}, {"value": "334", "qualifiers": ["A"], "dateTime": "2008-05-27T18:30:00.000-05:00"}, {"value": "333", "qualifiers": ["A"], "dateTime": "2008-05-27T18:45:00.000-05:00"}, {"value": "334", "qualifiers": ["A"], "dateTime": "2008-05-27T19:00:00.000-05:00"}, {"value": "336", "qualifiers": ["A"], "dateTime": "2008-05-27T19:15:00.000-05:00"}, {"value": "334", "qualifiers": ["A"], "dateTime": "2008-05-27T19:30:00.000-05:00"}, {"value": "334", "qualifiers": ["A"], "dateTime": "2008-05-27T19:45:00.000-05:00"}, {"value": "334", "qualifiers": ["A"], "dateTime": "2008-05-27T20:00:00.000-05:00"}, {"value": "333", "qualifiers": ["A"], "dateTime": "2008-05-27T20:15:00.000-05:00"}, {"value": "334", "qualifiers": ["A"], "dateTime": "2008-05-27T20:30:00.000-05:00"}, {"value": "333", "qualifiers": ["A"], "dateTime": "2008-05-27T20:45:00.000-05:00"}, {"value": "336", "qualifiers": ["A"], "dateTime": "2008-05-27T21:00:00.000-05:00"}, {"value": "334", "qualifiers": ["A"], "dateTime": "2008-05-27T21:15:00.000-05:00"}, {"value": "338", "qualifiers": ["A"], "dateTime": "2008-05-27T21:30:00.000-05:00"}, {"value": "333", "qualifiers": ["A"], "dateTime": "2008-05-27T21:45:00.000-05:00"}, {"value": "331", "qualifiers": ["A"], "dateTime": "2008-05-27T22:00:00.000-05:00"}, {"value": "336", "qualifiers": ["A"], "dateTime": "2008-05-27T22:15:00.000-05:00"}, {"value": "334", "qualifiers": ["A"], "dateTime": "2008-05-27T22:30:00.000-05:00"}, {"value": "336", "qualifiers": ["A"], "dateTime": "2008-05-27T22:45:0
```

Comment:
Please don't use real urls when testing? I'm sorry I didn't get that from you question earlier. You should use a fake url (that is something like 'http:fakenwisservice.org') and then always mock.